config/database.yml

```diff
@@ -5,8 +5,6 @@ development:
   adapter: postgresql
   database: theodinproject
   host: localhost
-  username: postgres
```

Comment:
This file shouldn't ever be changed. This will cause a lot of problems for anyone else who tries to run the app locally if this is committed.

Can we remove this from the PR please?